backend/FwLite/FwDataMiniLcmBridge/Api/FwDataMiniLcmApi.cs

```diff
@@ -896,7 +894,8 @@ internal void AddComplexFormComponent(ILexEntry lexComplexForm, ComplexFormCompo
 
     internal void InsertComplexFormComponent(ILexEntry lexComplexForm, ICmObject lexComponent, BetweenPosition<ComplexFormComponent>? between = null)
     {
-        var entryRef = lexComplexForm.ComplexFormEntryRefs.SingleOrDefault();
+        //there could be multiple valid refs, but we have no way of selecting which one to use so just use the first as that's what LCM does
+        var entryRef = lexComplexForm.ComplexFormEntryRefs.FirstOrDefault();
```

Comment:
It would be an improvement to prefer a `ComplexFormEntryRef` that contains a component referenced by one of the `between` components.